mmpose/datasets/pipelines/bottom_up_transform.py

```diff
@@ -6,19 +8,68 @@
 from .shared_transform import Compose
 
 
+def get_warpmatrix(theta, size_input, size_dst, size_target):
+    """Calculate the transformation matrix under the constraint of unbiased.
+    Paper ref: Huang et al. The Devil is in the Details: Delving into Unbiased
+    Data Processing for Human Pose Estimation (CVPR 2020).
+
+    Args:
+        theta (float): Rotation angle in degrees.
+        size_input (np.ndarray): Size of input image [w, h].
+        size_dst (np.ndarray): Size of output image [w, h].
+        size_target (np.ndarray): Size of ROI in input plane [w, h].
+
+    Returns:
+        matrix (np.ndarray): A matrix for transformation.
+    """
+    size_target = size_target
```

Comment:
What does this line do

Reply:
it is meaning less now, i will check and remove this